### ISSUE TITLE
English manual translation

### ISSUE THREAD

**Issue by czchmate** [osoumen/C700#2]
I translated the manual to English as best as I could. Here is the html file.

I saved it as a TXT file so that I could upload it. Just change the file type back to html and it should load fine.
[how_to_use_the_c700_(eng).txt](https://github.com/osoumen/C700/files/427100/how_to_use_the_c700_.eng.txt)

**Comment by osoumen** [comment]
@czchmate Great!
What this document we may be distributed bundled and plug-ins ?

**Comment by czchmate** [comment]
You can distribute it.

Here are the 32-bit and 64-bit VST plug-ins compiled in the latest Visual Studio 2015 with Windows 10.0.10586.0 set as the target platform.

**Comment by czchmate** [comment]
Sorry. Here we go again. Just rename .txt to .dll and these plug-ins should work.

[C700.txt](https://github.com/osoumen/C700/files/430460/C700.txt)
[C700_64.txt](https://github.com/osoumen/C700/files/430461/C700_64.txt)

**Comment by osoumen** [comment]
@czchmate
Thank you. In , at the time of the next version up , it will also distribute English document .

Current binary of c700 is built with Visualstudio2013. What this is there was something of compatibility problems caused ?
I am , if it is necessary , use the VisualStudio2015.

**Comment by czchmate** [comment]
Line 23 of C700_Prefix_VST.pch needs to be:
#if _MSC_VER < 1900
    #define snprintf _snprintf
#endif

In Lines 522 and 523, the abs function needs to be fabs to utilize the double buf:

```
    if (max < fabs(buf[i]))
        max=fabs(buf[i]);
```

After those changes, Visual Studio 2015 should compile correctly.

**Comment by osoumen** [comment]
I agree. I wanted to know why do we need to build in Visual Studio 2015 the c700.
Since I do not know much about Windows, and binary built with Visual Studio 2013, I do not know what is the difference in the build was a binary in Visual Studio 2015.

**Comment by akirna** [comment]
@czchmate, thanks for the translation!
